x-pack/plugins/monitoring/public/application/pages/page_template.tsx

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { useTitle } from '../hooks/use_title';
+
+interface PageTemplateProps {
+  title: string;
+  children: React.ReactNode;
+}
+
+export const PageTemplate = ({ title, children }: PageTemplateProps) => {
```

Comment:
This will probably evolve as the migration for the [Main directive](x-pack/plugins/monitoring/public/directives/main/index.js)